nrf-hal-common/src/monotonic.rs

```diff
@@ -1,13 +1,46 @@
 /*!
 Implements the [Monotonic](rtic_monotonic::Monotonic) trait for the TIMERs and the RTCs.
 
-<Här borde vi ha användings exempel>
+<Strong> Example (RTC): </Strong>
+```rust
+// RTC0 with a frequency of 32 768 Hz
+type MyMono = MonotonicRtc<RTC0, 32_768>;
 
-< Här borde vi ha docs om RTC>
+// Make sure lfclk is started
+let clocks = hal::clocks::Clocks::new(cx.device.CLOCK);
+let clocks = clocks.start_lfclk();
+
+let mono = MyMono::new(cx.device.RTC0, &clocks).unwrap();
+```
+
+<Strong> Example (Timer): </Strong>
+```rust
+// TIMER0 with a frequency of 16 000 000 Hz
+type MyMono = MonotonicTimer<TIMER0, 16_000_000>;
+let mono = MyMono::new(cx.device.TIMER0);
+
+```
+
+A simple example using the timer/rtc can be found under `nrf-hal/examples/monotonic-blinky`
```

Comment:
Replace actual link

```md
A simple example using the timer/rtc can be found under the nrf-hal [examples](https://github.com/nrf-rs/nrf-hal/tree/master/examples/monotonic-blinky).
```
though this link is broken now, if the feature is merged it will be correct.